.drone.star

```diff
@@ -23,7 +23,7 @@ OC_UBUNTU = "owncloud/ubuntu:20.04"
 # Todo: update or remove the following images
 # https://github.com/owncloud/client/issues/10070
 OC_CI_CLIENT_FEDORA = "owncloudci/client:fedora-39-amd64"
-OC_CI_SQUISH = "owncloudci/squish:fedora-39-7.2.1-qt66x-linux64"
+OC_CI_SQUISH = "owncloudci/squish:fedora-39-8.0.0-qt67x-linux64"
```

Comment:
we should update the squish image in the next PR after this PR gets merged and squish-image is built. I will update it
```suggestion
OC_CI_SQUISH = "owncloudci/squish:fedora-39-7.2.1-qt66x-linux64"
```